JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -15,479 +15,456 @@
+  scaleTable$showSpecifiedColumnsOnly <- TRUE
+  scaleTable$position <- 1
+  
+  # Add columns to table
+  scaleTable$addColumnInfo(name = "case", title = "", type = "string")
+  if (options$meanScale)
+    scaleTable$addColumnInfo(name = "mu", title = "mean", type = "number")
+  if (options$sdScale)
+    scaleTable$addColumnInfo(name = "sd", title = "sd", type = "number")
+  if (options$mcDonaldScale)
+    scaleTable$addColumnInfo(name = "omega", title = "McDonald's \u03C9",     
+                             type = "number")
+  if (options$alphaScale)
+    scaleTable$addColumnInfo(name = "alpha", title = "Cronbach's \u03B1",
+                             type = "number")
+  if (options$gutmannScale)
+    scaleTable$addColumnInfo(name = "lambda", title = "Gutmann's \u03BB6",
+                             type = "number")
+  if (options$glbScale)
+    scaleTable$addColumnInfo(name = "glb", title = "Greatest lower bound",
+                             type = "number")
+  if (options$averageInterItemCor)
+    scaleTable$addColumnInfo(name = "rho", title = "Average interitem correlation",
+                             type = "number")
+  if (options$confAlpha){
+    overTitle <- paste0(100*options$confAlphaLevel, "% Confidence Interval")
+    scaleTable$addColumnInfo(name = "lower", title = "Lower", type = "number", 
+                             overtitle = overTitle)
+    scaleTable$addColumnInfo(name = "upper", title = "Upper", type = "number", 
+                             overtitle = overTitle)
+  }
+  
+  if (options$missingValues == "excludeCasesListwise") 
+    exclwise = " listwise"
+  else 
+    exclwise = " pairwise"
+  
+  variables <- unlist(options$variables)
+  nObs      <- nrow(dataset)
+  nExcluded <- sum(!complete.cases(dataset))
+  nValid    <- nObs - nExcluded
+  
+  if(length(variables) >= 2) {
+    message <- sprintf("Of the observations, %d were used, %d were excluded%s, 
+                     and %d were provided.", nValid, nExcluded, exclwise, nObs)
+    if (options$glbScale && length(variables) == 2)
+      message <- paste(message, "Warning: Greatest lower bound can 
+    only be calculated for three or more variables.")
+    scaleTable$addFootnote(message, symbol = "<em>Note.</em>")
+  }
+  
+  jaspResults[["scaleTable"]] <- scaleTable
+  
+  # Compute/get the results
+  relyFit <- .reliabilityComputeResults(jaspResults, dataset, options, ready)
+  
+  res <- try(.reliabilityScaleFill(jaspResults, dataset, options, relyFit, ready))
+  if(isTryError(res))
+    scaleTable$setError(.extractErrorMessage(res))
+}
 
+.reliabilityItemTable <- function (jaspResults, dataset, options, ready) {
+  if (!(options$alphaItem || options$gutmannItem || options$itemRestCor ||
+        options$meanItem  ||options$mcDonaldItem || options$sdItem))
+    return()
+  
+  if (!is.null(jaspResults[["itemContainer"]][["table"]])) 
+    return()
+  
+  jaspResults[["itemContainer"]] <- createJaspContainer("Item Statistics")
```

Comment:
There can only be one item statistics table, right? Because then we don't need the container around it.

Reply:
Will I remove the container then? I was trying to keep the layout the same as before

Reply:
Ah, yeah, I don't really understand why we have 
-> Item Statistics
--> Item Reliability Statistics

I think the outer container doesn't add any information and can just be removed to be inline with other analyses